src/QsCompiler/QirGeneration/QIR/RuntimeLibrary.cs

```diff
@@ -27,13 +27,11 @@ public static class RuntimeLibrary
 
         // to-string
         public const string BigIntToString = "bigint_to_string";
-        public const string BoolToString = "bool_to_string";
         public const string DoubleToString = "double_to_string";
         public const string IntToString = "int_to_string";
-        public const string PauliToString = "pauli_to_string";
         public const string QubitToString = "qubit_to_string";
-        public const string RangeToString = "range_to_string";
-        public const string ResultToString = "result_to_string";
+        public const string RangeToString = "range_to_string"; // todo: eliminate
```

Comment:
Do these `todo`s need to be addressed before merging this case? If they will be kept for this change, it would be good to comment on when/why the strings need to be eliminated.

Reply:
No, this is a todo related to a spec update - or rather, it is possible (and likely reasonable) to not relay on the runtime for this functionality, but also lower priority compared to the general fixes in this PR.